@shared/api/helpers/extensionMessaging.ts

```diff
@@ -19,11 +18,7 @@ export const sendMessageToContentScript = (msg: {}): Promise<Response> => {
     { source: EXTERNAL_MSG_REQUEST, messageId: MESSAGE_ID, ...msg },
     window.location.origin,
   );
-  return new Promise((resolve, reject) => {
-    if (!window.freighter) {
```

Comment:
We no longer have the ability to set `window.freighter` on dapps with manifest version 3